src/covizpy/plot_metric.py

```diff
@@ -1,24 +1,88 @@
-def plot_metric(
-    metric="positive_rate",
-    date_from=None,
-    date_to=None
-):
+def plot_metric(metric='positive_rate', date_from="2022-01-01", date_to="2022-01-13"):
     """
     Create a line chart visualizing COVID total new
     cases and another metric for a specific time period
     
     Parameters
     ----------
-    metric : str, optional
-        The name of the metric to be plotted with the new COVID cases. 
-        It can be one of the these: "reproduction_rate", "positive_rate", "cardiovasc_death_rate ",
-        or any other numeric column
+    metric    : str, optional
+                The name of the metric to be plotted with the new COVID cases. 
+                It can be one of the these: "reproduction_rate", "positive_rate",
+                or any other numeric column
     date_from : str, optional
-        Start date of the plot, by default "None" will use 2 weeks prior today's date
-    date_to : str, optional
-        End date of the plot, by default "None" will use today's date
+                Start date of the plot in "YYYY-MM-DD" format, by default "2022-01-01"
+    date_to   : str, optional
+                End date of the plot in "YYYY-MM-DD" format, by default "2022-01-13"
     Returns
     -------
     chart
         The line chart created
     """
+    
+    try:
+        covid_df = pd.read_csv('https://covid.ourworldindata.org/data/owid-covid-data.csv')
```

Comment:
see if you would make use of get_data to retrieve the df but is fine if stick to the current approach. If yes, then this statement can move after all the input checks.

Reply:
Great idea Thomas! Will use getdata() and move this to after the input checks.